presto-kudu/bin/run_kudu_tests.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -euo pipefail -x
+
+KUDU_MASTER_RPC_PORT=17051
+KUDU_TSERVER_RPC_PORT=17050
+KUDU_MASTER_WEB_PORT=17081
+KUDU_TSERVER_WEB_PORT=17080
+DOCKER_HOST=$(ip -4 addr show docker0 | grep -Po 'inet \K[\d.]+')
```

Comment:
does it work on macos?

Reply:
I don't understand what you mean with macros in this context. But maybe it is easier to understand now with docker-compose.